App/Screens/Home/Header/Header.tsx

```diff
@@ -86,3 +63,56 @@ const styles = StyleSheet.create({
     marginTop: scale(-2) // FIXME We shouldn't need that, with `alignItems: 'center'` on .distance
   }
 });
+
+export function Header(props: HeaderProps) {
+  const { api } = useContext(ApiContext);
+  const { currentLocation, isGps } = useContext(CurrentLocationContext);
+  const { onChangeLocationClick } = props;
+
+  const distanceUnit = i18n.t('distance_unit');
```

Comment:
Merge error. You need to keep `const shortDistanceUnit = localizedDistanceUnit('short');`